test/esp/resources/custom_signature_test.rb

```diff
@@ -190,7 +190,7 @@ class CustomSignatureTest < ActiveSupport::TestCase
           end
 
           should 'return errors' do
-            signature = ESP::CustomSignature.run(external_account_id: 999, regions: 'us_east_1', language: @custom_signature.language, signature: @custom_signature.signature)
+            signature = ESP::CustomSignature.run(external_account_id: 99999999999, regions: 'us_east_1', language: @custom_signature.language, signature: @custom_signature.signature)
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).